types/manager-types.yaml

```diff
@@ -807,6 +819,22 @@ node_types:
   manager.nodes.ManagerIPSetter:
     derived_from: cloudify.nodes.ApplicationModule
     properties:
+      os_user:
+        description: IP setter's OS username. Will be created if not existing, and used to run the service.
+        type: string
+        default: cfyipsetter
+      os_group:
+        description: IP setter's OS group. Will be created if not existing, and used to run the service.
+        type: string
+        default: cfyipsetter
+      os_homedir:
+        description: IP setter's OS user's home directory. Will not be created if missing.
+        type: string
+        default: /etc/cloudify
+      sudoers_include_dir:
```

Comment:
In general, please remember that if more users should be added to sudoers file, we should probably think of a better approach.. maybe as part of some setup node or some other idea.